catalog/plugin/catalog-plugin-metacard-validation/src/main/java/ddf/catalog/metacard/validation/MetacardValidityMarkerPlugin.java

```diff
@@ -90,6 +94,9 @@ public UpdateRequest process(UpdateRequest input)
                 counter.getOrDefault(Validation.VALIDATION_WARNINGS, 0),
                 counter.getOrDefault(Validation.VALIDATION_ERRORS, 0));
 
+        INGEST_LOGGER.info("{} metacards were removed for failing an enforced validator.",
```

Comment:
@stustison I disagree, I think this should be info. It's just 1 line per ingest and it can be _very_ confusing when everything you thought you ingested doesn't get ingested for whatever reason. Even having the separate logs can be confusing, because not everyone knows about the ingest log, and I think sometimes the ingest command doesn't direct the user to the ingest log (much less say where it can be found)